ksp_plugin/plugin.cpp

```diff
@@ -267,15 +266,13 @@ void Plugin::EndInitialization() {
   // Log the serialized ephemeris.
   serialization::Ephemeris ephemeris_message;
   ephemeris_->WriteToMessage(&ephemeris_message);
-  std::string const bytes = ephemeris_message.SerializeAsString();
-  auto const hex = base::HexadecimalEncode(
-      base::Array<std::uint8_t const>(
-          reinterpret_cast<std::uint8_t const*>(bytes.data()), bytes.size()),
-      /*null_terminated=*/true);
+  auto const hex =
+      base::HexadecimalEncode(SerializeAsBytes(ephemeris_message).get(),
+                              /*null_terminated=*/true);
   // Begin and end markers to make sure the hex did not get clipped (this might
   // happen if the message is very big).
   LOG(INFO) << "Ephemeris at initialization:\nbegin\n"
-            << reinterpret_cast<char const*>(hex.data.get()) << "\nend";
+            << hex.data.get() << "\nend";
```

Comment:
Would fit on the previous line.

Reply:
Not fully; this is how clang-format folds it.